Software/Python/requirements.txt

```diff
@@ -1 +1,2 @@
 future
+quick2wire-api
```

Comment:
is quick2wire still required?

Reply:
Nope. Good catch. This is just a left-over. I'm deleting it.